tests/auto/gltfexporter/tst_gltfexporter.cpp

```diff
@@ -0,0 +1,198 @@
+        tmp.removeRecursively();
+        tmp.cdUp();
+    }
+    tmp.mkdir("kuesa-gltfexporter-test");
+    tmp.cd("kuesa-gltfexporter-test");
+    tmp.setFilter(QDir::Dirs | QDir::Files | QDir::NoDotAndDotDot);
+    return tmp;
+}
+} // namespace
+
+class tst_GLTFExporter : public QObject
+{
+    Q_OBJECT
+
+private Q_SLOTS:
+
+#if defined(KUESA_DRACO_COMPRESSION)
+    void checkBoxCompression()
+    {
+        SceneEntity scene;
+        GLTF2Context ctx;
+
+        GLTF2Parser parser(&scene);
+        parser.setContext(&ctx);
+
+        QString asset(ASSETS "Box.gltf");
+        QDir tmp = setupTestFolder();
+
+        QJsonObject new_asset;
+        // WHEN
+        {
+            auto res = parser.parse(asset);
+            QVERIFY(res != nullptr);
+
+            GLTF2ExportConfiguration configuration;
+            configuration.setMeshCompressionEnabled(true);
+            configuration.setAttributeQuantizationLevel(GLTF2ExportConfiguration::Position, 5);
+            configuration.setAttributeQuantizationLevel(GLTF2ExportConfiguration::Normal, 5);
+
+            GLTF2Exporter exporter;
+            exporter.setContext(&ctx);
+            exporter.setScene(&scene);
+            exporter.setConfiguration(configuration);
+
+            QFile asset_file(asset);
+            asset_file.open(QIODevice::ReadOnly);
+            new_asset = exporter.saveInFolder(
+                    QDir(ASSETS),
+                    tmp);
+        }
+
+        // THEN
+        {
+            QVERIFY(!new_asset.empty());
+
+            ctx = GLTF2Context{};
+            auto res = parser.parse(QJsonDocument{ new_asset }.toJson(), tmp.absolutePath());
+            QVERIFY(res != nullptr);
+
+            QVERIFY(tmp.exists("compressedBuffer.bin"));
+            QVERIFY(!tmp.exists("Box0.bin"));
+            QCOMPARE(tmp.count(), 1U);
+        }
+    }
+
+    void checkCarCompression()
+    {
+        SceneEntity scene;
+        GLTF2Context ctx;
+
+        GLTF2Parser parser(&scene);
+        parser.setContext(&ctx);
+
+        QString asset(ASSETS "car/DodgeViper.gltf");
+        QDir tmp = setupTestFolder();
+
+        QJsonObject new_asset;
+        // WHEN
+        {
+            auto res = parser.parse(asset);
+            QVERIFY(res != nullptr);
+
+            GLTF2ExportConfiguration configuration;
+            configuration.setMeshCompressionEnabled(true);
+            configuration.setAttributeQuantizationLevel(GLTF2ExportConfiguration::Position, 5);
+            configuration.setAttributeQuantizationLevel(GLTF2ExportConfiguration::Normal, 5);
+
+            GLTF2Exporter exporter;
+            exporter.setContext(&ctx);
+            exporter.setScene(&scene);
+            exporter.setConfiguration(configuration);
+
+            QFile asset_file(asset);
+            asset_file.open(QIODevice::ReadOnly);
+            new_asset = exporter.saveInFolder(
+                    QDir(ASSETS),
+                    tmp);
+        }
+    }
```

Comment:
Can't we at least check something to verify the compression occured? It's lacking some QCOMPARE or QVERIFY in there. :-)

Reply:
done